src/main/kotlin/com/mutualmobile/praxisspringboot/services/orgs/UserProjectService.kt

```diff
@@ -9,4 +10,8 @@ interface UserProjectService {
         projectId: String,
         userId: String
     ): ResponseEntity<ApiResponse<HarvestUserProject>>
+
+    fun findUserProject(projectId: String, userId: String): HarvestUserProject?
```

Comment:
@shubhamsinghmutualmobile can we document this method, like what this is used for ..